pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -226,6 +226,26 @@ func classifyNodes(
 	return lowNodes, highNodes
 }
 
+func usage2KeysAndValues(usage map[v1.ResourceName]*resource.Quantity) []interface{} {
```

Comment:
```suggestion
func usageToKeysAndValues(usage map[v1.ResourceName]*resource.Quantity) []interface{} {
```

better to be consistent and not use number shortcuts in the names

Reply:
Done